test/test.snapshot.js

```diff
@@ -437,6 +440,10 @@ QUnit.test('Snapshot and text tracks', function(assert) {
   assert.equal(this.player.textTracks()[0].language, 'es');
   assert.equal(this.player.textTracks()[0].mode, 'disabled');
 
+  // Double check that the track remains disabled
+  assert.equal(this.player.remoteTextTracks()[0].mode, 'disabled');
+  assert.equal(this.player.textTracks()[0].mode, 'disabled');
```

Comment:
It looks like this assertion happens twice, previously on line 441. What could have changed it in that time?

Reply:
The original reason for the textTrackChangeHandler was to mitigate user agents changing the track mode during ad playback, I added this here to make sure that doesn't happen for browsers other than iOS but looks like I forgot to add a `clock.tick()`. Will fix that now!